app/views/layouts/application.html.erb

```diff
@@ -13,8 +13,19 @@
     https://github.com/calebthompson/title#usage
   %>
   <title><%= title %></title>
-  <link rel="preload" as="font" type="font/woff2" crossorigin href="/assets/WorkSans-Regular.woff2">
-  <link rel="preload" as="font" type="font/woff2" crossorigin href="/assets/WorkSans-Bold.woff2">
+  <link rel="preload"
+        as="font"
+        type="font/woff2"
+        crossorigin
+        href=<%= asset_path("WorkSans-Regular.woff2") %>
```

Comment:
I think you need quotes around the `<`,`>`.

Reply:
Good 🎣. I don't seem to need the quotes but I suspect that's because the browser is covering for the mistake.